examples/plotting/file/color_scatter.py

```diff
@@ -3,7 +3,7 @@
 
 .. bokeh-example-metadata::
     :apis: bokeh.plotting.Figure.scatter
-    :refs: :ref:`userguide_tools`
+    :refs: :ref:`userguide_bokehjs` > :ref:`userguide_bokehjs_interfaces_plotting`
```

Comment:
@karen-poon The `userguide_bokehjs` ref goes to the JavaScript docs. Since this is a Python example, that won't be useful for users. I would also like to keep the reference to the "tools" section. This example adds *lots* tools, so that was the motivation for adding a reference there. 

I'd suggest this (untested, will need to confirm it works):

```suggestion
    :refs: :ref:`userguide_tools`, :ref:`userguide_plotting` > :ref:`userguide_plotting_scatter_markers`
```